service/src/test/java/bio/terra/profile/service/profile/ProfileServiceUnitTest.java

```diff
@@ -0,0 +1,148 @@
+package bio.terra.profile.service.profile;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.ArgumentMatchers.anyInt;
+import static org.mockito.ArgumentMatchers.anyString;
+import static org.mockito.ArgumentMatchers.eq;
+import static org.mockito.Mockito.doThrow;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+
+import bio.terra.common.exception.ForbiddenException;
+import bio.terra.common.exception.UnauthorizedException;
+import bio.terra.common.iam.AuthenticatedUserRequest;
+import bio.terra.profile.common.BaseUnitTest;
+import bio.terra.profile.db.ProfileDao;
+import bio.terra.profile.model.CloudPlatform;
+import bio.terra.profile.service.iam.SamService;
+import bio.terra.profile.service.iam.model.SamAction;
+import bio.terra.profile.service.iam.model.SamResourceType;
+import bio.terra.profile.service.job.JobBuilder;
+import bio.terra.profile.service.job.JobMapKeys;
+import bio.terra.profile.service.job.JobService;
+import bio.terra.profile.service.profile.flight.ProfileMapKeys;
+import bio.terra.profile.service.profile.flight.create.CreateProfileFlight;
+import bio.terra.profile.service.profile.flight.delete.DeleteProfileFlight;
+import bio.terra.profile.service.profile.model.BillingProfile;
+import java.time.Instant;
+import java.util.List;
+import java.util.Optional;
+import java.util.UUID;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.mockito.Mock;
+
+public class ProfileServiceUnitTest extends BaseUnitTest {
```

Comment:
This PR has some basic unit tests for `ProfileService` and associated flights. I need to add more unit and connected tests, however. Thinking of doing that in a separate PR.